htsinfer/infer_read_orientation.py

```diff
@@ -22,5 +21,45 @@ def infer(
         LIBTYPE string according to Salmon documentation, cf.
         https://salmon.readthedocs.io/en/latest/library_type.html
     """
+    #create .tmp directory in "inputDir" and retuns .tmp path
+    tmpDir = mkTmpDir(inputDir)
+    
+    # call functions of issues #25, 26, 27 and 34
+    
+    # delete .tmp
+    rmTmpDir(tmpDir)
+        
     # implement logic
     return "U"
+    
+
+def mkTmpDir(inputDir):
+    """Function to create a temporary directory for the storage of 
+    temporary data."""
+  
+    import os
```

Comment:
Please put imports at the top of the module, right below the module's docstring. FYI, the same [Python style guidelines](https://www.python.org/dev/peps/pep-0008/#imports) mentioned above also suggest an order in which imports should be listed, with standard/builtin packages first, followed by third party libraries (the ones we have to install) and then the imports of modules from this very package (i.e., `htsinfer`), and with each group (builtin, third party, current package) being separated by an empty line. For example:

```py
import os  # this is a builtin library, we don't have to install it specifically with PyPI
import sys  # also builtin

import Bio  # BioPython is a third party library, it needs to be specifically installed

from htsinfer import infer_read_orientation  # this is from the current package
```

Specifically, if you move the `import os` to the beginning of the file, you need to group it together with the import from `typing` which is also builtin, and within each group it's good practice to follow alphabetical order of the package name:

```py
import os
from typing import Union
```